src/Behat/Symfony2Extension/Driver/KernelDriver.php

```diff
@@ -21,8 +22,12 @@
  */
 class KernelDriver extends BrowserKitDriver
 {
-    public function __construct(KernelInterface $kernel, $baseUrl = null)
+    public function __construct(KernelInterface $kernel, $baseUrl = null, $shared = false)
     {
-        parent::__construct($kernel->getContainer()->get('test.client'), $baseUrl);
+        $client = (true === $shared) ?
+            new Client($kernel) :
```

Comment:
`Client` is now deprecated. Should be replaced with `Symfony\Component\HttpKernel\HttpKernelBrowser`